assets/components/circlesIntroduction/circlesIntroduction.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
```

Comment:
Is this one a Shared component? It looks like you are only using it in support-landing-page.

Reply:
For now I am, it will be in many pages though =).